docs/api-gateway-v2/10-02-connect-api-rule.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Cannot connect to a service exposed by an APIRule
+type: Troubleshooting
+---
+
+##  Basic troubleshooting
+
+API Gateway is a controller. It adds a status to the rules it processes. For basic troubleshooting, check the APIRule status:
+
+   ```bash
+   kubectl describe apirules.gateway.kyma-project.io -n {NAMESPACE} {NAME}
+   ```
+
+If the status is `Error`, edit the APIRule and fix issues described in `.Status.APIRuleStatus.Desc`. If you still encounter issues, make sure the API Gateway, Hydra and Oathkeeper are running or take a look at other, more specific troubleshooting guides.
```

Comment:
I would link here this document https://kyma-project.io/docs/master/components/api-gateway-v2/#custom-resource-api-rule-status-codes